src/lib/components/SiteFooter/index.jsx

```diff
@@ -350,6 +404,13 @@ SiteFooter.defaultProps = {
   showMobileLinks: true,
   appleAppStoreUrl: null,
   googlePlayUrl: null,
+  supportedLanguages: [],
+  languageForm: {
+    screenReaderLabel: 'Choose Language',
+    submitLabel: 'Apply',
```

Comment:
i18n here too.

Reply:
@albemarle have you got an example of how to i18n a component's defaultProps (given that it doesn't have access to the ```intl``` object)?

Reply:
It seems like ```react-intl``` may not support translating defaultProps (https://github.com/yahoo/react-intl/issues/834)

Reply:
Discussed with @albemarle and ```react-intl``` doesn't currently support defaultProps (so removing them for now and moving to required props).